src/DataTypes/Serializations/ISerialization.cpp

```diff
@@ -217,6 +277,10 @@ String getNameForSubstreamPath(
             stream_name += ".sparse";
         else if (it->type == Substream::SparseOffsets)
             stream_name += ".sparse.idx";
+        else if (it->type == Substream::ReplicatedElements)
```

Comment:
Myabe `".repl"` and `".repl.idx"` to make consistent with sparse?
Or `"repl.data"` and `"repl.idx"` just to make shorter.